en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,54 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+Sometimes, there is no easy way to render the value to the backoffice user so we have added
+a ``stringValue`` to ``ContentAttribute``.
+The ``stringValue`` is a projection of the ``value`` in a string.
+
+Transformers
+------------
+
+Adding a transformer
+~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra transforms the value into a string thanks to transformer implementing the interface
+``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``.
+Different transformer are already available in ``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+For the service to be recognised as a transformer, it should use the tag
+``open_orchestra_backoffice.value_transformer.strategy`` :
+
+.. code-block:: yml
+
+    open_orchestra_backoffice.value_transformer.array:
+            class: %open_orchestra_backoffice.value_transformer.array.class%
+            tags:
+                - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Then implement methods in your transformer :
+
+- ``transform`` : transform value to an HTML string
+- ``support`` : return true if the transformer must be used
+- ``getName`` : use by manager as index for the transformer
```

Comment:
"used by the manager as an index"
however i don't think you should speak about the manager here, the interface defines a name as an identifier but doesn't have a knowledge about the transformer.